src/database/config.ts

```diff
@@ -0,0 +1,7 @@
+export const config = {
+  database: process.env.PGDATABASE || "db",
```

Comment:
these defaults don't make much sense, the server should simply print an error and exit if any of these vars are not set